src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx

```diff
@@ -801,7 +719,7 @@
     <value>PackageSourcesListBox</value>
   </data>
   <data name="&gt;&gt;PackageSourcesListBox.Type" xml:space="preserve">
-    <value>NuGet.Options.PackageSourceCheckedListBox, NuGet.PackageManagement.UI, Version=0.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35</value>
+    <value>NuGet.Options.PackageSourceCheckedListBox, NuGet.PackageManagement.UI, Version=6.1.0.32767, Culture=neutral, PublicKeyToken=31bf3856ad364e35</value>
```

Comment:
This happened automagically by saving the file of the control. I would guess it's wrong, since it needs to be a sliding version.  ..?

Reply:
Was able to install this on a clean VM, and had no issue loading the options dialog (where the assembly version wouldn't exist).
![image](https://user-images.githubusercontent.com/49205731/147958920-4a35d004-f27e-4357-9de2-e807ee0e3986.png)

Reply:
Maybe you can manually edit to revert version value here, not sure if regress anything at all.

Reply:
I agree with @erdembayar 

There's one more change. 

I'd be afraid that this fails when it's not a local build.

Reply:
Given the concern is that CI can't build this properly, and as I mentioned I already tested the VSIX (from CI) on a clean VM with no issues, I believe this comment can be resolved.